docs/docs/shards/index.md

```diff
@@ -164,4 +164,41 @@ Showing how we mix the Clojure-like language as a template language
     [info] [2021-04-04 21:05:50.123] [T-285088] [logging.cpp::53] [counter-10] 20
     ```
 
+## Shards' types
+
+This section documents all the valid data types that are accepted by various shards either as their input value or as their parameter values. These data types also apply to the output created by any shard.
+
+Valid data types for every shard are listed under the `Type` column of their Parameters, Input, and Output sections (types are enclosed within parenthesis and if multiple types apply then they are separated by a space).
+
+!!! note
+    Types are helpful as they reduce errors in programming. They are also very useful in visual programming as type-matching can be used to reduce the dropdown options when prompting a user on what shard to use next (depending on which shard's input type matches with the current shard's output type).
+
+### Any
+
+Indicates that all data types are valid.
+
+For example, the type `Any` is accepted for the `:Value` parameter (and for the 'Input' field) in `(AnyNot)`. This means that all data types listed in this section are valid for a comparison by `(AnyNot)`.
+
+=== "Code"
+
+    ```clojure linenums="1"
+    (AnyNot
+    :Value [(Any)]
```

Comment:
this code won't work...  not sure what is it

Reply:
Sorry, that was meant to be just the shards function-defintion. Fixed it and added working code samples for both `Any` and `None`.
Fixed in commit [#04463b000aa936607b03463a14bb4fafbb160841](https://github.com/fragcolor-xyz/shards/pull/325/commits/04463b000aa936607b03463a14bb4fafbb160841)